src/beautifiers/prettydiff.coffee

```diff
@@ -48,6 +48,7 @@ module.exports = class PrettyDiff extends Beautifier
     EJS: true
     HTML: true
     Handlebars: true
+    Nunjucks: true
```

Comment:
Pretty Diff is using `auto` by default, which will likely work. However, there should also be a `case` below for handling `Nunjucks` and mapping to the appropriate Pretty Diff language, such as `html`.